source/tyr/migrations/versions/061f7f29d76d_add_ridesharingservice_table.py

```diff
@@ -0,0 +1,45 @@
+"""Add RidesharingService table
+
+Revision ID: 061f7f29d76d
+Revises: 5127fcffb248
+Create Date: 2020-10-07 15:13:00.279814
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '061f7f29d76d'
+down_revision = '5127fcffb248'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        'ridesharing_service',
+        sa.Column('created_at', sa.DateTime(), nullable=False),
+        sa.Column('updated_at', sa.DateTime(), nullable=True),
+        sa.Column('id', sa.Text(), nullable=False),
+        sa.Column('klass', sa.Text(), nullable=False),
+        sa.Column('discarded', sa.Boolean(), nullable=False),
```

Comment:
What about a default value : server_default='false' like for 'args'. As it is initialized in the class, it's preferable but not compulsory.

Reply:
ok